app/test/e2e/mocks.js

```diff
@@ -0,0 +1,11 @@
+const nock = require('nock');
+
+const createMockUser = (mockUser, persist = false) => {
+    let scope = nock(process.env.CT_URL);
+    if (persist) scope = scope.persist();
```

Comment:
this is evil-ish, as it hides away how many times this mock is actually called. typically you want to use `.times()` or an equivalent instead

Reply:
Your commits removed the need from persisting the nock scope!